.github/workflows/push-triggers.yml

```diff
@@ -0,0 +1,38 @@
+name: Push-Triggers
+
+.anchor-branches-tags: &branches-tags
+  branches:
+    - main
+    - develop
+    - qa-develop
+    - "release-**"
+  tags:
+    - "*"
```

Comment:
Instead of tags, please use release
``` 
  release:
    types: [published]
  pull_request:
    types: [opened, reopened, synchronize]
    paths:
      - 'charts/**'
  ```